cmd/edit/machinepool/machinepool.go

```diff
@@ -18,11 +18,9 @@ import (
 var machinePoolKeyRE = regexp.MustCompile(`^[a-z]([-a-z0-9]*[a-z0-9])?$`)
 
 func editMachinePool(cmd *cobra.Command, machinePoolID string, clusterKey string, cluster *cmv1.Cluster,
-	r *rosa.Runtime) {
-	var err error
+	r *rosa.Runtime) error {
 	if !machinePoolKeyRE.MatchString(machinePoolID) {
-		r.Reporter.Errorf("Expected a valid identifier for the machine pool")
-		os.Exit(1)
+		return fmt.Errorf("expected a valid identifier for the machine pool")
```

Comment:
@chenz4027 Please correct me if I'm wrong, but I thought we did this before, only to get feedback from QE that the error messages should be capitalized? I would suggest we don't change the output of messages here and ignore the warning from Go.

Reply:
@robpblake Sure, I'll move them back to capitalized string